src/index.ts

```diff
@@ -90,7 +97,7 @@ export function parseForESLint(
             errors: rootAST.errors,
         }
         const templateBody =
-            template != null && templateLang === "html"
+            template != null && (templateLang === "html" || templateLang === "pug")
                 ? Object.assign(template, concreteInfo)
                 : undefined
```

Comment:
Maybe we should pass a `templateLang` also to the `templateBody` so the plugin can easily find out if it's html or pug.
But on the other side this can be easily be checked with checking the `<template *>` part